<launch>
  <arg name="base_frame" default="kr8_r1420_rcb/link_6"/>
  <arg name="tool0_frame" default="kr8_r1420_rcb/welder_tool_surface"/>
  <arg name="num_samples" default="4"/>

  <node name="tool_calibration_node" pkg="tool_point_calibration" type="tool_calibration" output="screen">
    <param name="base_frame" value="$(arg world)"/>
    <param name="tool_frame" value="$(arg tool_frame)"/>
    <param name="num_samples" value="$(arg num_samples)"/>
  </node>
</launch>
